addons/l10n_latam_invoice_document/models/account_move.py

```diff
@@ -193,13 +193,16 @@ def _check_invoice_type_document_type(self):
 
     def _get_l10n_latam_documents_domain(self):
         self.ensure_one()
+        internal_types = [False]
```

Comment:
You're right, Selection fields are False when emptied (NULL in postgres). I was thinking about Char fields. I changed it in my PR as well.

Reply:
In the EC localization, it has the "internal type" filter twice, but it is not a risk scenario for us. Could be an improvement section to adapt to this code.
![image](https://github.com/odoo/odoo/assets/78817305/6bde8791-516d-450b-a5d4-fd076eed45bc)